scripts/bert/bert_qa_dataset.py

```diff
@@ -76,7 +77,8 @@ def preprocess_dataset(dataset, transform, num_workers=8):
             if data:
                 dataset_transform.extend(data)
 
-        dataset = SimpleDataset(dataset_transform)
+        dataset = SimpleDataset(dataset_transform).transform(
+            lambda x: (x[0], x[1], x[2], x[3], x[4], x[5]))
```

Comment:
when SQuAD=2.0, would this be `lambda x: (x[0], x[1], x[2], x[3], x[4], x[5], x[6]))`?

Reply:
No, `x[0], x[1], x[2], x[3], x[4], x[5]` are `example_ids, inputs, token_types, valid_length, start_label, end_label`